src/barplot.ts

```diff
@@ -247,6 +247,15 @@ export function drawLabels(size:number[], data:INumericalVector, labels: d3.Sele
     'display': (rowHeight >= 15) ? 'inline' : 'none',
     'font-size': '14px'
   });
+  const categories = (<any>data.desc.value).categories;
+  const nameToLabel = new Map<string, string>();
+  if (categories) {
```

Comment:
So you mean `data` should be of type `IAnyVector`?
So it should look like this: `export function drawLabels(size:number[], data:IAnyVector, labels: d3.Selection<any>)`

In this case, a cast is still needed since `IValueTypeDesc` neither has a `categories` field. Isn't it?